src/Icons/src/Icon.php

```diff
@@ -155,6 +155,14 @@ public function toHtml(): string
                 continue;
             }
 
+            if ('fill' === $name) {
+                $this->innerSvg = str_replace('fill="currentColor"', "fill=\"{$value}\"", $this->innerSvg);
+            }
+
+            if ('stroke' === $name) {
+                $this->innerSvg = str_replace('stroke="currentColor"', "stroke=\"{$value}\"", $this->innerSvg);
```

Comment:
Let's keep an only place we echo value and espace / encode them correctly (just below)